### ISSUE TITLE
importing cache/reuse data from another .nextflow/work dir

### ISSUE THREAD

**Issue by notestaff** [nextflow-io/nextflow#3558]
TL;DR: add ability to import, into the current work directory and its corresponding .nextflow dir, cache/reuse data from another .nextflow/work dir.

Often, I start a long-running Nextflow pipeline execution and then work on some addition to the pipeline while the old one runs.  To avoid concurrency issues, I do that work on a separate checkout of the pipeline repo.  However, in that checkout, I lose access to cache/reuse information from the main checkout, and vice versa.  If permitting concurrent execution of two pipelines in the same directory is too complicated, a next-best solution would be to allow cache/reuse information generated in one pipeline checkout to be imported into (or used directly from) another pipeline checkout.
